lib/charms/haproxy/v0/haproxy_route.py

```diff
@@ -844,7 +919,7 @@ def provide_haproxy_route_requirements(
         self.update_relation_data()
 
     # pylint: disable=too-many-arguments,too-many-positional-arguments,too-many-locals
```

Comment:
Is too many locals still need to be ignored ?

Reply:
Doesn't seem to, will remove it in a follow-up PR, thanks.